test/functional/rpc_listaccounthistory.py

```diff
@@ -72,6 +73,28 @@ def run_test(self):
         for txs in results:
             assert(hasattr(txs['amounts'], '__len__') and (not isinstance(txs['amounts'], str)))
 
+        # list {"maxBlockHeight":103, "txn":1}, should list without blockheight = 103, txn=2. i.e without MintToken
+        results = self.nodes[0].listaccounthistory(collateral_a, {"maxBlockHeight":103, "txn":1})
+        #Note(surangap): the results here has 2 extra send receive entries with different txids. not sure where they come from.
```

Comment:
@bvbfan any idea where these extra entries have come from?

Reply:
`Auto Auth` is used to ensure auth on specific address.